message.go

```diff
@@ -2,7 +2,18 @@ package log
 
 import (
 	"fmt"
+	"os"
 	"time"
+
+	"golang.org/x/term"
+)
+
+var ResetColor = "\033[0m"
```

Comment:
I don't like the inconsistency of having this definition separated from the others.

I think I get your intent to separate the ANSI-functional "reset" name from the log-logical "level" names below, but if so then I think they should all be done that way.

E.g., define `AnsiReset` and `AnsiRed` and so forth and then define `DebugColor` to `AnsiRed` and `InfoColor` to `AnsiReset` to clarify your separation and make the code a bit more flexible and readable.

Reply:
That's fair. I only pulled out the default/reset because I simplified the function to have only a single return, and with the default set as a constant I couldn't override it in the else statement, and I tried to take the minimal code path.